components/Collections/src/private/MDCCollectionViewStyler.m

```diff
@@ -561,8 +562,8 @@ - (UIImage *)backgroundImageForCellLayoutAttributes:(MDCCollectionViewLayoutAttr
   // Get cell color.
   UIColor *backgroundColor = _cellBackgroundColor;
   if ([_delegate respondsToSelector:@selector(collectionView:cellBackgroundColorAtIndexPath:)]) {
-    UIColor *customBackgroundColor =
-        [_delegate collectionView:_collectionView cellBackgroundColorAtIndexPath:attr.indexPath];
+    UIColor *customBackgroundColor = [_delegate collectionView:_collectionView
+                                cellBackgroundColorAtIndexPath:attr.indexPath];
```

Comment:
is this correct?

Reply:
This is aligning the colons, so it makes sense if the method invocation starts on the preceding line.